opennms-services/src/main/java/org/opennms/netmgt/vacuumd/AutomationProcessor.java

```diff
@@ -706,9 +709,10 @@ public boolean runAutomation() throws SQLException {
         if (hasTrigger()) {
             LOG.debug("runAutomation: {} trigger statement is: {}", m_automation.getName(), m_trigger.getTriggerSQL());
         }
-            
+
```

Comment:
Could you please revert the whitespace/formatting changes on lines 712, 715, 726. Thanks